### PR TITLE
New breathtaking medical stack: the mourning poultice!

### PR DESCRIPTION
**Pull request by Krysonism**
## About The Pull Request
This pr adds a new type of medical stack, obtainable from botany: the mourning poultice.

These poultices convert 10 brute and 10 burn into oxy per application, and can be used on corpses as well as the living.

It takes 20u bungotoxin, 20u aloe juice and 20u cellulose to create one stack.

I have split all the existing medical stacks into their own dmi, stacks_medical.dmi.

image

## Why It's Good For The Game
This addition compliments some of the  less used botany healing tools, such as strange reagent and space tobacco.

Botany has a long standing theme of producing premium healing items and reagents. I think we should encourage more altruistic botany strategies, especially considering the current state of super death plants.

read more:
https://hackmd.io/83Gj479fQWqDbGjFCxRP6Q

## Changelog
:cl:
add: Botany can create a new type of medical stack: mourning poultices. This item converts brute and burn into oxy and can even be used on corpses.
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
youre breathtaking!

**Comment by Krysonism** [comment]
The sounds are both cut from public domain sounds obtained from freesounds.

I am very unfamiliar with audacity but hopefully they sound okay.

**Comment by bluetshirtguy** [comment]
I love this, keep em coming so i can be a medieval doctor!

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 1]
```diff
@@ -182,3 +184,10 @@
 	for(var/i = 1, i <= created_volume, i++)
 		new /obj/item/stack/medical/mesh/advanced(location)
 
+/datum/chemical_reaction/poultice
+	required_reagents = list(/datum/reagent/toxin/bungotoxin = 20, /datum/reagent/cellulose = 20, /datum/reagent/consumable/aloejuice = 20)
+
+/datum/chemical_reaction/poultice/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
```
Comment (Rohesie) [inline code/modules/reagents/chemistry/recipes/medicine.dm:192]:
```suggestion
	for(var/i = 1 in i to created_volume)
```

**Review by Twaticus — APPROVED** [review]

**Pull request merged**